src/assets/js/eventregistration.js

```diff
@@ -3,8 +3,13 @@ import { proto } from './lib/trace_location_pb';
 import { encode } from 'uint8-to-base64';
 import Cleave from 'cleave.js';
 import Papa from 'papaparse';
+import $ from 'jquery';
 import 'regenerator-runtime/runtime'
 
```

Comment:
Please use the existing tab javascript:
https://github.com/corona-warn-app/cwa-website/blob/master/src/assets/js/app.js#L233-L241

Reply:
Ok, done as you suggested